service/service.py

```diff
@@ -302,6 +302,42 @@ def update_wishlists(wishlist_id):
     app.logger.info("Wishlist with ID [%s] updated.", wishlist_id)
     return make_response(jsonify(message), status.HTTP_200_OK)
 
+######################################################################
+# UPDATE AN EXISTING ITEM IN A WISHLIST
```

Comment:
Professor said he wouldn't allow updating product names or product ids. We have nothing to update then. Please remove this method.

Reply:
I have removed the API for now